integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -119,6 +119,28 @@ resources:
     {{- end }}
 {{- end -}}
 
+{{- define "alluxio.journal.format.resources" -}}
```

Comment:
Have a separate section for this journal formatting job. The resource should be different from a normal master container.